apps/session-manager/.node-version

```diff
@@ -0,0 +1 @@
+20.10.0
```

Comment:
the monorepo has `20.12.2` as node version. could we also put it here?

Reply:
Done in [[#IOPID-1763] add eslint + fix based on code review](https://github.com/pagopa/io-auth-n-identity-domain/pull/1/commits/95787ff45a3ddbfeccbb0f616623810c957e883f)